### DISCUSSION TITLE
Ayuda

### DISCUSSION THREAD

**Discussion by Max80318** [Launch-X-Latam/MisionFrontEnd#50] [General]
Como entro alas clases en linea de launchx

O todo será en forma teorica no abra clases vir3

**Comment by Max80318** [comment]
*Clases virtuales

**Comment by ManyPH** [comment]
Las sesiones serán los miercoles y viernes a las 7:00 pm (Hora Ciudad de México -> GMT-5) a través de Teams. 
Para entrar solamente tienes que hacer click en el enlace que envía el mission comander el mismo dia de la sesión. La publicación aparece en Teams y también se envía un mensaje al correo.
Para acceder a las sesiones grabadas: #22

**Comment by Max80318** [comment]
Buenas noches ya me registre en teams

> Message ID:
> ***@***.***
> .com>
>
Y no me llega ningún correo para entrar ala clase
Tampoco tengo comander

**Reply by Amhedriel** [reply]
Buenas

No te llega correo, para entrar a la clase tienes que estar el Miércoles y Viernes a la hora que te dijo Many.

El Commander o Sherpa es el mismo para todos

**Reply by Max80318** [reply]
Que tal ya me llego el correo para unirme a al equipo de teams gracias

Ahora solo tengo una duda tienen algun bloqueo de servidores de Internet porque quise conectarme al live de ayer y no me cargo

**Reply by Amhedriel** [reply]
Me parece que no tienen bloqueo de servidores, a menos que vivas en China, Venezuela, Corea del Norte o similares.

Pero en lo personal, el live de ayer casi no pude ver video, al menos el audio estaba bien.

**Reply by Amhedriel** [reply]
Pero revisa las grabaciones y deberías poder ver todo lo que se habló ayer sin problemas, al ser grabación no debería haber problemas de conexión lenta o simil.

**Reply by Max80318** [reply]
Aya
Ps ami ni audio
Me da solo queda en cargando
Y soy de Guatemala

**Reply by Amhedriel** [reply]
Sera tu velocidad¿?

**Reply by Amhedriel** [reply]
Aunq puede ser también la antigüedad de los equipos, mi PC tiene como 10 años y creo que por eso no me mostraba bien la imagen, mi velocidad de internet es de 300 MB/s asi que la velocidad no creo que sea mi problema, sin embargo, puede que la velocidad del Commander haya tenido intermitencia o algo parecido, lo que vale ahora es que los videos grabados se puedan ver con más detenimiento, probemos suerte en el directo de mañana, saludos.

**Reply by JhonFre7** [reply]
A todos nos paso que se lageaba el video pero el audio andaba bien

**Comment by Max80318** [comment]
Me podrias ayudar como hago para que me envien el enlace

**Reply by Amhedriel** [reply]
El enlace lo verás momentos antes de entrar al live, lo podrás ver en Microsoft Teams - Equipos - Canales _ General.
Ahí verás las publicaciones que son importantes, Archivos, Grabaciones, en donde verás las grabaciones de las clases de live.

**Comment by Max80318** [comment]
Esos dias no se recibe clase virtual??

**Reply by Amhedriel** [reply]
Las Clases serán solamente los Miércoles y los Viernes, este Lunes fue una excepción a modo de presentación eh inicio de actividades
